### PR TITLE
Markdown format

### LINKED ISSUE #303: Including images into announcement
### Steps to reproduce
1. This is not an issue, rather it is feature request. I really liked announcement and have been using it a lot. But what I really like to suggest here is that adding a possibilities to attach images in addition to texts on detailed paragraph. 
2. I think this one is already asked here, it would really really great if there is editing features for the texts part, like font size, fonts, font styles and many. 

Thanks a lot for the great app. 
### Server configuration

**Operating system**:
Fedora Server 33
**Web server:**
HTTP
**Database:**
Postgresql 
**PHP version:**
7.4.14
**Nextcloud version:** (see Nextcloud admin page)
20.0.8

### PR DESCRIPTION
**Pull request by tcitworld**
Use the editor from the text app to write announcements, and make markdown rendering in backend.

Closes #303 #644

### CODE REVIEW

File: .github/contributing.md
Comment (nickvergessen) [inline .github/contributing.md:1]: what happened here?

File: src/AdminSettings.vue [hunk 1]
```diff
@@ -20,28 +20,41 @@
 -->
 
 <template>
-	<NcSettingsSection :title="t('announcementcenter', 'Announcements')">
-		<NcSettingsSelectGroup id="announcementcenter_admin_group"
-			v-model="adminGroups"
-			:label="t('announcementcenter', 'These groups will be able to post announcements.')"
-			@input="updateGroups" />
+	<NcSettingsSection :name="t('announcementcenter', 'Announcements')">
+		<div>
```
Comment (nickvergessen) [inline src/AdminSettings.vue:24]: Anything but this div is changed whitespaces? Could it be avoided to keep the diff small and reduce conflict with other work?

File: src/App.vue
Comment (nickvergessen) [inline src/App.vue:1]: All changes in here are whitespace changes? Could it be avoided to keep the diff small and reduce conflict with other work?

**Review by nickvergessen — COMMENTED** [review]
This makes the Text app a requirement, which kind of contradicts with my idea of the app. I prefer it to be very minimalistic and e.g. last week we disabled Text and then announced that which would no longer be possible.

It boils down/goes back to https://github.com/nextcloud/announcementcenter/pull/725#pullrequestreview-1925033930

File: src/AdminSettings.vue (hunk 1 above)
Comment (tcitworld) [reply src/AdminSettings.vue:24]: Indentation, attribute `:title` becomes  `:name` for `NcSettingsSection`, and the label for `NcSettingsSelectGroup` is extracted.

**Comment by nickvergessen** [comment]
I'm closing this as per https://github.com/nextcloud/announcementcenter/pull/812#pullrequestreview-2135258762
I don't see this app adding so much complexity.
It can be merged if someone is taking over the ownership.

**Comment by tcitworld** [comment]
It's fine. We're maintaining this for the foreseeable future at https://framagit.org/framasoft/framaspace/custom-apps/announcementcenter/

**Pull request closed without merging**